eyeshade/workers/acvote.js

```diff
@@ -18,40 +20,44 @@ module.exports = (runtime) => {
         runtime.captureException(e, { extra: { topic: voteTopic, message } })
         continue
       }
+
       await insertVote(runtime, date, vote, client)
     }
   })
 }
 
 module.exports.insertVote = insertVote
 
-async function insertVote (runtime, date, vote, client) {
-  const surveyorId = date + '_' + vote.fundingSource
-  const cohort = 'control'
-  const tally = vote.voteTally
-  const voteValue = vote.baseVoteValue
-  const publisher = vote.channel
+async function insertVote (runtime, date, vote, client, hasValidCountryFunc = hasValidCountry) {
+  // Check if votes are for valid country
+  if (await hasValidCountryFunc(runtime, vote.channel)) {
+    const surveyorId = date + '_' + vote.fundingSource
+    const cohort = 'control'
+    const tally = vote.voteTally
+    const voteValue = vote.baseVoteValue
+    const publisher = vote.channel
 
-  const surveyorUpdate = `
+    const surveyorUpdate = `
       insert into surveyor_groups (id, price, virtual) values ($1, $2, true)
       on conflict (id) do nothing;
       `
-  await runtime.postgres.query(surveyorUpdate, [
-    surveyorId,
-    voteValue
-  ], client)
+    await runtime.postgres.query(surveyorUpdate, [
```

Comment:
I wonder which postgres library it's used here

Reply:
`pg`